unit/util/interval/comparisons.cpp

```diff
@@ -0,0 +1,208 @@
+/*******************************************************************\
+ Module: Unit tests for variable/sensitivity/abstract_object::merge
+ Author: DiffBlue Limited
+\*******************************************************************/
+
+#include <testing-utils/use_catch.h>
+
+#include <util/arith_tools.h>
+#include <util/interval.h>
+#include <util/std_expr.h>
+#include <util/std_types.h>
+#include <util/symbol_table.h>
+
+#define V(X) (bvrep2integer(X.get(ID_value).c_str(), 32, true))
+#define V_(X) (bvrep2integer(X.c_str(), 32, true))
+#define CEV(X) (from_integer(mp_integer(X), signedbv_typet(32)))
```

Comment:
:no_entry_sign: I believe there are existing functions thats to do this, and will make it clearer, but this is incomprehensibke

Reply:
at the very least:
V -> int_value
CEV -> int_expr
and delete the unused V_